app/notification_push_test.go

```diff
@@ -1425,6 +1425,7 @@ func TestAllPushNotifications(t *testing.T) {
 }
 
 func TestPushNotificationRace(t *testing.T) {
+	th := Setup(t).InitBasic()
```

Comment:
`.InitBasic()` is not required I think. And we also need a `defer th.Teardown()`